### ISSUE TITLE
extension with config data raises an error.

### ISSUE THREAD

**Issue by sander76** [MadLittleMods/markdown-icons#8]
@MadLittleMods Just checking: Has this ever worked?

```python
md = markdown.Markdown(extensions=['iconfonts(prefix=mypref-)'])
```

Adding the `prefix=mypref-` raises an error with the latest version of markdown.

**Comment by MadLittleMods** [comment]
@sander76 There are tests for it, https://github.com/MadLittleMods/markdown-icons/blob/d707cc413754854221920249d1820e54dfc845ad/tests/test_iconfonts.py

**Comment by sander76** [comment]
@MadLittleMods I know. I ran them. They failed when you include the extra config data inside quotes. If you look at the markdown docs it doesn't show the option. That's why I was wondering: could it be it is not supported anymore? Thinking further, should the current code be considered legacy and start a new plugin (or new version) supporting only the current version of markdown and python version>3.4 ?

**Comment by MadLittleMods** [comment]
@sander76 Yes, it was only tested in older versions of Markdown and is why I asked you if it worked here, https://github.com/MadLittleMods/markdown-icons/issues/6#issuecomment-438792448

**Comment by sander76** [comment]
@MadLittleMods I guess I was a bit quick on the conclusion. I have been using your plugin for a couple of years, but only with the "basic" configuration which works just fine. After I found out it worked for me I thought all was fine not really looking at the other options the plugin provides.... 🙄.

So not sure how to proceed now...

**Comment by sander76** [comment]
@MadLittleMods Sorry about this

**Comment by sander76** [comment]
@MadLittleMods I am totally fine to create a trimmed down version of your plugin and giving you all credits. I don't think I can live up to the expectation of both keeping backwards compatibility, upgrading to the new markdown version and keeping all config options intact....

**Comment by MadLittleMods** [comment]
@sander76 We can create a new v3 branch for use with the latest Python markdown version

Since it is a major version, it can have breaking changes
